ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/NETWORK_PORTS_TESTS.yml

```diff
@@ -141,6 +141,35 @@ network_ports:
       channel_id: 43
       mode: "active"
 
+  # Tests below test hostname and platform matching.
+  # Test regex match all switches and specific platform match criteria.
+  - switches:
+      - network-ports-tests.*
+    switch_ports:
+      - Ethernet47-48
+    platforms:
+      - 720XPM-48Y6
+    profile: pc
+    endpoint: PCs
+
+  # Test regex match switch and no match on platform.
+  - switches:
+      - network-ports-tests.2
+    switch_ports:
+      - Ethernet47-48
```

Comment:
network-ports-tests.1 already has 47,48. Please add any other port numbers.

Reply:
this is only matching `network-ports-tests.2`